test/Templates.Test/Helpers/WebDriverFactory.cs

```diff
@@ -24,13 +26,20 @@ public static IWebDriver CreateWebDriver()
             var result = IsAppVeyor ? CreateFirefoxDriver() : CreateEdgeDriver();
             result.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(1);
             return result;
+
+            IWebDriver CreateFirefoxDriver()
+            {
+                return WebDriverFactory.CreateFirefoxDriver(new FirefoxOptions(){
+                    AcceptInsecureCertificates = true
+                });
+            }
         }
 
         private static IWebDriver CreateEdgeDriver()
             => new EdgeDriver(EdgeDriverService.CreateDefaultService(BinDir));
```

Comment:
I haven't found a way to ignore SSL certificate errors on Edge, but trusting the certificate on your box makes it work just fine.

Reply:
Would it work to automate clicking the "ignore invalid cert" button in the browser, similar to how it's suggested here? https://github.com/angular/protractor/issues/2377#issuecomment-289920884

Reply:
@SteveSandersonMS Yeah, I saw that option when I was looking into it, but I wanted to avoid going that route as feels "fragile". The easiest thing to do is to manually trust the cert on your environments if you can.